### PR TITLE
feature(Plugins/Premium/Fromat Painter -> PowerPaste): Update docs for premium plugins section from formatpainter to powerpaste

### PR DESCRIPTION
**Pull request by hamza0867**
Related Ticket: TINY-8595

Description of Changes:
This PR makes updates to premium plugins docs from format painter to power paste. This PR is a bit big, but it contains a lot of small changes to premium plugins that did not need much to update ( with the exception of power paste ).

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

**Comment by lnewson** [comment]
It looks like there's a fair few options that need an id added on their heading, so to save commenting on each one I'm just going to leave the single action this time.

- [ ] Add ids to configuration option headings (e.g. `partials/configuration/formats.adoc`)

**Comment by lnewson** [comment]
In `modules/ROOT/partials/configuration/tiny_pageembed_classes.adoc` the `Disable select` heading level is incorrect which causes a ToC issue. It needs one more `=` so it should be this instead:
```
==== Disable select
```

File: modules/ROOT/partials/configuration/mentions_item_type.adoc [hunk 1]
```diff
@@ -7,7 +8,7 @@ This option sets which user interface item type to use when displaying the list
 * The `name` item will only display the user's name.
 * The `profile` item will display the user's name and can display an optional image and description.
 
-For information on the properties required for the user object provided to <<mentions_fetch,`mentions_fetch`>>, see: <<userproperties,User properties>>.
+For information on the properties required for the user object provided to xref:mentions_fetch[`mentions_fetch`], see: xref:userproperties[User properties].
```
Comment (lnewson) [inline modules/ROOT/partials/configuration/mentions_item_type.adoc:11]:
```suggestion
For information on the properties required for the user object provided to xref:plugins/premium/mentions.adoc#mentions_fetch[`mentions_fetch`], see: xref:userproperties[User properties].
```

File: modules/ROOT/partials/configuration/mentions_fetch.adoc [hunk 2]
```diff
@@ -1,6 +1,7 @@
+[[mentions_fetch]]
 === `mentions_fetch`
 
-This option lets you request a list of users from your server that match a search query. The callback gets passed two parameters: one is the search query object, the other is the success callback to execute with the results. The query object has a term property that contains what the user has typed after the "@" sign. The success call should contain an array of users. For information on the user properties to pass the success callback for the available mentions item types (`mentions_item_type`), see: <<userproperties,User properties>>.
+This option lets you request a list of users from your server that match a search query. The callback gets passed two parameters: one is the search query object, the other is the success callback to execute with the results. The query object has a term property that contains what the user has typed after the "@" sign. The success call should contain an array of users. For information on the user properties to pass the success callback for the available mentions item types (`mentions_item_type`), see: xref:userproperties[User properties].
```
Comment (lnewson) [inline modules/ROOT/partials/configuration/mentions_fetch.adoc:4]:
```suggestion
This option lets you request a list of users from your server that match a search query. The callback gets passed two parameters: one is the search query object, the other is the success callback to execute with the results. The query object has a term property that contains what the user has typed after the "@" sign. The success call should contain an array of users. For information on the user properties to pass the success callback for the available mentions item types (`mentions_item_type`), see: xref:plugins/premium/mentions.adoc#userproperties[User properties].
```

File: modules/ROOT/partials/configuration/mentions_item_type.adoc [hunk 3]
```diff
@@ -26,9 +27,10 @@ tinymce.init({
 });
 ----
 
+[[userproperties]]
 ==== User properties
 
-The following table describes the properties available for user objects provided to the <<mentions_fetch,`mentions_fetch` callback>>. Properties may be required, optional, or not available; depending on the <<mentions_item_type,`mentions_item_type`>> and <<mentions_select,`mentions_select`>> options.
+The following table describes the properties available for user objects provided to the xref:mentions_fetch[`mentions_fetch` callback]. Properties may be required, optional, or not available; depending on the xref:mentions_item_type[`mentions_item_type`] and xref:mentions_select[`mentions_select`] options.
```
Comment (lnewson) [inline modules/ROOT/partials/configuration/mentions_item_type.adoc:33]:
```suggestion
The following table describes the properties available for user objects provided to the xref:plugins/premium/mentions.adoc#mentions_fetch[`mentions_fetch` callback]. Properties may be required, optional, or not available; depending on the xref:mentions_item_type[`mentions_item_type`] and xref:plugins/premium/mentions.adoc#mentions_select[`mentions_select`] options.
```

File: modules/ROOT/partials/configuration/mentions_menu_hover.adoc [hunk 4]
```diff
@@ -39,7 +39,7 @@ tinymce.init({
 
 include::partial$misc/requires_5_9v.adoc[]
 
-If `mentions_menu_hover` is resolved with an object specifying the type and user details, a predefined hover card template will be used. To use the predefined template, set `type` to `'profile'`. For details on the user properties required for the `profile` template, see: <<userproperties,User properties>>.
+If `mentions_menu_hover` is resolved with an object specifying the type and user details, a predefined hover card template will be used. To use the predefined template, set `type` to `'profile'`. For details on the user properties required for the `profile` template, see: xref:userproperties[User properties].
```
Comment (lnewson) [inline modules/ROOT/partials/configuration/mentions_menu_hover.adoc:42]:
```suggestion
If `mentions_menu_hover` is resolved with an object specifying the type and user details, a predefined hover card template will be used. To use the predefined template, set `type` to `'profile'`. For details on the user properties required for the `profile` template, see: xref:plugins/premium/mentions.adoc#userproperties[User properties].
```

File: modules/ROOT/partials/configuration/mentions_select.adoc [hunk 5]
```diff
@@ -48,7 +49,7 @@ tinymce.init({
 
 include::partial$misc/requires_5_6v.adoc[]
 
-If `mentions_select` is resolved with an object specifying the type and user details, a predefined hover card template will be used. To use the predefined template, set `type` to `'profile'`. For details on the user properties required for the `profile` template, see: <<userproperties,User properties>>.
+If `mentions_select` is resolved with an object specifying the type and user details, a predefined hover card template will be used. To use the predefined template, set `type` to `'profile'`. For details on the user properties required for the `profile` template, see: xref:userproperties[User properties].
```
Comment (lnewson) [inline modules/ROOT/partials/configuration/mentions_select.adoc:52]:
```suggestion
If `mentions_select` is resolved with an object specifying the type and user details, a predefined hover card template will be used. To use the predefined template, set `type` to `'profile'`. For details on the user properties required for the `profile` template, see: xref:plugins/premium/mentions.adoc#userproperties[User properties].
```

**Review by lnewson — APPROVED** [review]

**Review by EkimChau — APPROVED** [review]

**Comment by hamza0867** [comment]
> In `modules/ROOT/partials/configuration/tiny_pageembed_classes.adoc` the `Disable select` heading level is incorrect which causes a ToC issue. It needs one more `=` so it should be this instead:
> 
> ```
> ==== Disable select
> ```

Done

**Pull request merged**